### PR TITLE
Change single quetos to double quetos in template 'Gemfile'

### PR DESCRIPTION
**Pull request by ch3rub1m**
Change single quetos to double quetos in template 'Gemfile' for complying with Style/StringLiterals 'double_quotes'.

### CODE REVIEW

**Comment by maclover7** [comment]
cc @fxn

File: railties/lib/rails/generators/rails/plugin/templates/Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
```
Comment (rafaelfranca) [inline railties/lib/rails/generators/rails/plugin/templates/Gemfile]: Should not this be double quotes too?

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

**Comment by rafaelfranca** [comment]
Is the application template already using this pattern?

**Comment by ianfleeton** [comment]
The generated templates do not attempt to conform to the Rails codebase's Rubocop quoting rules. For example, the generated Gemfile favors single quotes.

Also, ```**/templates/**/*``` is excluded in .rubocop.yml which matches this file.

If you ignore the exclusion then only the code generating code should conform to the double quoting Rubocop rules, which this PR does not address.

**Comment by ch3rub1m** [comment]
Ok, @ianfleeton, Thank you for your explanation, but why not keep code style consistent between codebase and templates? I think this is not elegant enough.

**Comment by fxn** [comment]
As a rule of thumb, In general we try to generate code which is consistent with our style, and follow also that style in documentation examples. The most notorious case of this is probably indented method definitions after `private` in scaffolds.

I wanted to contribute this remark to clarify this point. The exclusion in RuboCop rules was probably added to not transfer our unification pass to end users touching too many templates at once (I am speculating, don't actually know the reason).

@rafaelfranca already reviewed, I support whatever he decides.

File: railties/lib/rails/generators/rails/plugin/templates/Gemfile (hunk 1 above)
Comment (ch3rub1m) [reply railties/lib/rails/generators/rails/plugin/templates/Gemfile]: yes, I think this should be double quotes too

Comment (ch3rub1m) [reply railties/lib/rails/generators/rails/plugin/templates/Gemfile]: but I deleted the fork so I can't change it now. Should I open a new pull request if you agree about this change?

**Pull request closed without merging**